WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubViewModel.swift

```diff
@@ -144,6 +144,15 @@ private extension AnalyticsHubViewModel {
                                             trailingDelta: orderValueDelta.string,
                                             trailingDeltaColor: Constants.deltaColor(for: orderValueDelta.direction))
     }
+
+    static func productCard(currentPeriodStats: OrderStatsV4?, previousPeriodStats: OrderStatsV4?) -> AnalyticsProductCardViewModel {
+        let itemsSold = StatsDataTextFormatter.createItemsSoldText(orderStats: currentPeriodStats)
+        let itemsSoldDelta = StatsDataTextFormatter.createOrderItemsSoldDelta(from: currentPeriodStats, to: previousPeriodStats)
```

Comment:
It looks like the stats are flipped here — the delta should be calculated from the previous period to the current period:

```suggestion
        let itemsSoldDelta = StatsDataTextFormatter.createOrderItemsSoldDelta(from: previousPeriodStats, to: currentPeriodStats)
```

Reply:
Crap, thanks for catching this!